src/test/java/bio/overture/ego/controller/UserControllerTest.java

```diff
@@ -241,12 +251,287 @@ public void createUser_EmailAlreadyExists_Conflict() {
             .preferredLanguage(randomEnum(LanguageType.class))
             .firstName(randomStringNoSpaces(10))
             .lastName(randomStringNoSpaces(10))
+            .identityProvider(GITHUB)
+            .providerId("github123")
+            .build();
+
+    // Create the user and assert OK
+    createUserPostRequestAnd(r).assertOk();
+  }
+
+  // TODO: add this test in when email required configuration is implemented
+  @Ignore
+  @Test
+  public void createUser_NoEmail_OK() {
+    val nonExistentProviderId = generateNonExistentProviderId(userService);
+    // Create a request without an email provided
+    val r =
+        CreateUserRequest.builder()
+            .status(randomEnum(StatusType.class))
+            .type(randomEnum(UserType.class))
+            .preferredLanguage(randomEnum(LanguageType.class))
+            .firstName(randomStringNoSpaces(10))
+            .lastName(randomStringNoSpaces(10))
+            .identityProvider(GITHUB)
```

Comment:
randomize this via `EntityGenerator.randomEnumExcluding(IdProviders.class, UNKNOWN)`